stories/google-logout.stories.js

```diff
@@ -37,6 +37,6 @@ story.add('Default Button', () => (
     </GoogleLogout>
   ),
   {
-    jest: ['google-logout.test.js', 'google-logout-dist.test.js'],
+    jest: ['google-logout.test.js', 'google-logout-dist.nottest.js'],
```

Comment:
you can remove `, 'google-logout-dist.nottest.js'`

Reply:
Removed